2019 midyear update (#1068)
writethedocs/www: docs/conf/cfp/east-2020.rst

Write the Docs East 2020 Call for Venues
========================================

We're also excited to start planning another US conference in 2020.

> The goal will be to make closer to folks on the east coast,

Reviewer: Perhaps "The goal is to make it closer to..." instead? I think "it" is missing after "make"

but we haven't figured out exactly where to host it yet.

> The plan is to host it sometime in the fall,

Author: ```suggestion
We're thinking of hosting it sometime in the fall,
```

> so that it is nicely spaced out with our Portland conference.

Author: ```suggestion
to give folks who want to attend both a chance to, and so things don't get too hectic for the organizing teams. 
```

Requirements
------------

> We have some lose requirements for venues that we're looking for.

Reviewer: change "lose" to "loose" -- I also think you can eliminate the rest of the sentence after "venues"

We have some loose requirements for venues.

It's OK if a venue doesn't fit one or two of these,

> but they are the general goal of what we're looking for.

Reviewer: Maybe "...but this is a general picture of what we're looking for." instead?

> * Fits up to 250 people

Reviewer: period after "people"

got to have a couple nitpicks :D

>     - Preferably with some tables along with chairs, but not a requirement

Reviewer: period after "requirement"

> * Has character. We are quite unlikely to try and use a conference center or hotel basement, as that is really not the vibe we're going for.

Reviewer: Maybe change the second sentence to:

We are unlikely to use a conference center or hotel space--it's not the vibe we're going for.

> * Has two or three additional rooms for an Unconference room (large or multiple small/medium), Staff/Green room (small), and Quiet room (small). The more additional rooms the better

Reviewer: period after "better"

* Not too expensive. Around or below $10,000 for the rental is a good starting point, but even better if it's free with a catering minimum or other deal.

Questions
---------

Here are some good questions to answer when you think about the venue:

* Is the venue easily reachable by public transit and/or does it have sufficient parking, depending on the transit options in your local area?

> * Is there affordable accommodations nearby? 

Reviewer: Change "Is" to "Are"

Are there affordable accommodations nearby?
